nvtabular/inference/triton/model.py

```diff
@@ -218,3 +216,52 @@ def get_hugectr_offsets(path):
         return np.cumsum(np.array(slot_sizes)).tolist()
     else:
         return None
+
+
+def _transform_tensors(input_tensors, column_group):
+    if column_group.parents:
+        tensors, kind = None, None
+        for parent in column_group.parents:
+            transformed, transformed_kind = _transform_tensors(input_tensors, parent)
+            if tensors is None:
+                tensors, kind = transformed, transformed_kind
+            else:
+                if kind != transformed_kind:
+                    # we have multiple different kinds of data here (dataframe/array on cpu/gpu)
+                    # we need to convert to a common format here first before concatentating.
+                    op = column_group.op
+                    target_kind = op.supports if op else Supports.CPU_DICT_ARRAY
+                    # note : the 2nd convert_tensors call needs to be stricter in what the kind is
+                    # (exact match rather than a bitmask of values)
+                    tensors, kind = convert_format(tensors, kind, target_kind)
+                    transformed, _ = convert_format(transformed, transformed_kind, kind)
+
+                _concat_tensors([tensors, transformed], kind)
+
+    else:
+        tensors = {c: input_tensors[c] for c in column_group.columns}
+        kind = Supports.CPU_DICT_ARRAY
+
+    if column_group.op:
+        try:
+            # if the op doesn't support the current kind - we need to convert
+            if not column_group.op.supports & kind:
+                tensors, kind = convert_format(tensors, kind, column_group.op.supports)
+
+            tensors = column_group.op.transform(column_group.columns, tensors)
+
+        except Exception:
```

Comment:
yeah, its to provide extra logging info (especially about *what* op failed: I found with some of the C++ optimized operators I'm going to introduce in the next PR it wasn't always clear since there is no python stack trace to inspect). The exception is re-raised immediately, so catching everything shouldn't be an issue 

Fun fact: without the 'raise' on line 255 pylint has the same objections to the catch being too broad as you and would have autofailed this PR with ```model.py:253:15: W0703: Catching too general exception Exception (broad-except)``` - but with the raise pylint is all good with it =).